metadata-ingestion/docs/sources/clickhouse/clickhouse_recipe.yml

```diff
@@ -35,4 +37,25 @@ source:
   config:
     host_port: localhost:9440
     scheme: clickhouse+native
-    secure: True
\ No newline at end of file
+    uri_opts:
+      secure: "true"
+
+#------------------------------------------------------------------------
+#Example: using ingestion with configured SSL-TLS and uri_opts example
+#See https://clickhouse.com/docs/en/guides/sre/configuring-ssl for connection details
+# ------------------------------------------------------------------------
+    #url form, prefered
```

Comment:
Please follow the other examples and mention the source type, config fields as well.